src/pages/contributing/product-development-lifecycle/index.mdx

```diff
@@ -0,0 +1,145 @@
+<AnchorLink>Definition of done</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+IBM teams follow a process framework called the Product Development Lifecycle (PDLC). IBM defines the PDLC as the process of taking a product capability from an idea through its development and release to market, and beyond. Since product development takes place in a continuously evolving market, the framework creates room for validation in every phase of the lifecycle.
+
+The PDLC is organized into three phases: discovery, delivery, and launch & scale. Once a product is in market, agile product teams do all these types of work concurrently, in parallel work streams, with feedback from in-market learnings shaping the priorities of both discovery and delivery.
+
+#### PDLC applied to contribution
+Carbon has applied the PDLC framework to how we expect new assets like components, features, and patterns to be contributed to Carbon. We have broken the contribution process up into the same three phases, each having their own set of success criteria.
+
+- **Discovery phase**: The purpose of the discovery phase is to address any major value, usability, feasibility and viability risks ahead of delivering product-quality code, in order to arrive at a successful product faster and at a lower cost. 
+- **Delivery phase**: The delivery phase emphasizes the disciplined execution of determining how we will realize user value and maximizing the efficiency of building it. This includes the functional and non-functional requirements, and the user experience required for customers to adopt product features within a production setting. 
+- **Launch and scale phase**: Launch and scale of course, involves having clearly defined success criteria or learning objectives that are monitored after launch. In this phase an asset will become stable and achieve Carbon’s “definition of done.”
+
+_For IBMers only_: You can read more about the PDLC on the [IBM Winning Products](https://w3.ibm.com/w3publisher/winning-products/how-we-work/product-development-lifecycle) website.
+
+<Row>
+<Column colLg={12}>
+
+![PDLC diagram](images/PDLC-01.png)
+
+</Column>
+</Row>
+
+## Discovery 
+
+The discovery phase is where research, exploration, and validation happen. This is when innovations to the system are proposed and reviewed.
+
+### Discovery criteria 
+
+For an enhancement or net new asset to warrant resourcing a discovery phase, we need to determine the requirements for an asset to be considered as an innovation. Proposals need to show that the component or pattern would be useful to many teams and unique to the system. 
+
+#### Key considerations:
+
+- Does it replicate anything in the system already, or is there truly a gap?
+- If the proposal does replicate an existing asset, is there evidence to show that the proposed solution is better?
+- Is there already an existing issue or proposal in [GitHub](https://github.com/carbon-design-system/carbon/issues) to address the gap?
```

Comment:
do you think it would be a good idea to link all the Carbon and Carbon for IBM Products repositories here? This way, people won't forget to check the other repos as well.

Additionally, would it be helpful to mention discussing the proposal or issue through Slack?